colour/__init__.py

```diff
@@ -908,6 +908,10 @@ def __getattr__(self, attribute):
             'colour.log_decoding_VLog',
             'colour.models.log_decoding_VLog',
         ],
+        [
+            'colour.log_decoding_FLog',
```

Comment:
Those API changes are not required here because the definitions have never been renamed or their access point changed: it is the first time they are featured in Colour.